### DISCUSSION TITLE
Welcome to ThoriumOS Discussions!

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/ThoriumOS#24] [Announcements]
## Welcome to ThoriumOS Discussions
 Here you can:
  * Ask questions (anything)
  * Share ideas.
  * Ask about building/installing ThoriumOS or ChromiumOS

    You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by Alex313031** [comment]
@jonsneyers The browser in ThoriumOS already has jpeg-xl support, but I also just added it system-wide via a package. This allows commandline usage, and should allow the CrOS media app the consume it for opening jxl images.

See > https://github.com/Alex313031/ThoriumOS/commit/f774b79c2930ae38ad5035adda693e8dc8f5fe4d

**Comment by 1Codealot** [comment]
Is there anyway to use thorium in a virtual machine?

**Comment by thelangosta** [comment]
For some reason the latest 126.xx version will not finish downloading. It starts and gets to about 50% and then the download stops and the error is that the "file wasn't available on site"
